<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<?pydev-testrun version="1.0"?><pydev-testsuite>
<summary errors="1" failures="0" finished="true" name="Test Run: 0" tests="0" total_time="Finished in: 0.00 secs."/>
<test location="/home/malik/workspace/Regression/Code/Regression.py" status="error" test="Code.Regression" time="0">
<stdout/>
<stderr><![CDATA[Traceback (most recent call last):
  File "/home/malik/.p2/pool/plugins/org.python.pydev_5.3.1.201610311318/pysrc/_pydev_runfiles/pydev_runfiles.py", line 468, in __get_module_from_str
    mod = __import__(modname)
  File "/home/malik/workspace/Regression/Code/Regression.py", line 3, in <module>
    import matplotlib.pyplot as plt
ImportError: No module named 'matplotlib'
ERROR: Module: Code.Regression could not be imported (file: /home/malik/workspace/Regression/Code/Regression.py).
]]></stderr>
</test>
<launch mode="debug">
<launch_memento><![CDATA[<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<launchConfiguration local="true" path="Regression Regression"/>
]]></launch_memento>
</launch>
</pydev-testsuite>
